CleverRaven/Cataclysm-DDA
# Remove a pointless door

#### Summary

SUMMARY: none

#### Purpose of change

One of the apartment layouts has a second story door that goes outside. I find it unlikely that the architect would anticipate the need for jumpimg from the second story down to the pavement below.

#### Describe the solution

Replace the door with a wall

#### Describe alternatives you've considered

I have not considered any alternatives

#### Testing

1. Test that a new game can be started
2. Use the overmap editor to place the s_apt special
3. Check that there is a wall where the door used to be

#### Additional context

none
